crates/starknet-devnet-core/src/state/mod.rs

```diff
@@ -153,7 +153,7 @@ pub struct StarknetState {
     rpc_contract_classes: Arc<RwLock<CommittedClassStorage>>,
     /// - initially `None`
     /// - indicates the state hasn't yet been cloned for old-state preservation purpose
-    historic_state: Option<DictState>,
+    historic_state: DictState,
```

Comment:
I don't remember exactly but think it was important that this was an `Option`. If indeed it is not needed anymore, the `/// documentation` need to be changed.

Reply:
I think the remaining documentation line was referring to the deleted one.

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/6b52603d2225e828b072cb54639d14f3d45683ec